src/main/java/com/oltpbenchmark/util/ResultWriter.java

```diff
@@ -99,10 +99,12 @@ public void writeConfig(PrintStream os) throws ConfigurationException {
     }
 
     public void writeSummary(PrintStream os) {
-        Map<String, Object> summaryMap = new TreeMap<>();
+        Map<String, Object> summaryMap = new LinkedHashMap<>();
```

Comment:
Cosmetic: try to keep the insertion order of the hash map when we output the json

Reply:
Although ... it doesn't appear to be working as expected. The `entrySet()` returned in the `JSONUtil` code that prints this looks all out of order unfortunately.

Reply:
Looked into this - it's because we use JSONObject to pretty print the output, which converts it back into a map and removes the ordering.